src/content/basics/secured_access_to_clusters/index.md

```diff
@@ -0,0 +1,79 @@
++++
+title = "Secured access to clusters for users and Giant Swarm support"
+description = "This documentation explains security measures for users and Giant Swarm support accessing offered services"
+date = "2019-05-09"
+weight = 45
+type = "page"
+categories = ["basics"]
++++
+
+# Secured access to clusters for users and Giant Swarm support
+
+In order to provide best service possible besides you having access to your clusters, also Giant Swarm staff will be able to connect.
+Thus it should be explained how this access differs and what security measures are brought into place so that your data is save with us. 
+
+## Intro
+Access to Giant Swarm clusters can be split into two parts. 
+
+1. Admin Access - designated for Giant Swarm staff for managing/development/support purposes.
+
+2. User Access - designated for Giant Swarm customers to interact with the offered services.
+
+If you would like to know more on different parts of infrastructure you can view the operational layer article [here](https://docs.giantswarm.io/basics/giant-swarm-operational-layers/)
+
+## Admin Access
+
+Admin access is guarded by Virtual Private Network(VPN) that is managed via certificates (public/private keys) and secured connection between both VPNs.
+Certificates management is handled by one of Giant Swarm private components. 
+The general principle is that certificate is configured for each individual Giant Swarm staff member.
+
+VPN Secured access points:
+
+* *SSH* - SSH Admin access is based on the Github SSO. Only users residing in the Github Giant Swarm Organization are allowed to authenticate.  
+  Following process describes more in detail the procedure of SSH authentication:
+  ![](./ssh_access_process.png)  
+  SSH access to Control Plane allows Giant Swarm also to manage and connect to underlying Data Platforms of the Customer.
+
+* *Control Plane Kubernetes API* - Usage of Kuberentes API on Control Plane also follows the authentication principles of the SSH connection. 
+The Github SSO based on Giant Swarm Organization is used to grant access. 
+
+***General VPN Connection Schema***
+
+Following schema illustrates how the VPN connection looks in practice. 
+
+![](./site-to-site-vpn.png)
+
+Giant Swarm uses two different VPN providers in order to bring best and fastest support possible.
+
+***Cloud Provider Access***
+
+At the current stage internal Giant Swarm operator components responsible for managing clusters lifecycle at Customer's Cloud 
+Provider are granted the admin rights by the customer to the given API.
```

Comment:
Why there is a line break here? I think can be your IDE, let me know if you need help to set it up

Reply:
oh, was breaking the line to stay within the wrap of 120 chars. Guess that's my python legacy